package.json

```diff
@@ -64,6 +64,7 @@
   },
   "dependencies": {
     "classnames": "^2.1.2",
+    "history": "^1.17.0",
```

Comment:
My bad @madeinfree I just pushed a commit with the requested changes.
@AllenFang Thank you for your hard work! Really looking forward to stable version of 3.0!